src/services/aws/pricing/function/GetProducts.ts

```diff
@@ -20,28 +21,31 @@ const createFilters = (filtersMap: Map<string, string>) => {
   return filters;
 };
 
-export const getProducts = async (filtersMap: Map<string, string> ): Promise<Product[]> => {
+export const getProducts = async (
+  filtersMap: Map<string, string>,
+  priceClient: PricingClient = client
```

Comment:
where is `client` variable coming from?

Reply:
sorry just seen it is imported from client-pricing

Reply:
👍